### ISSUE TITLE
Failed to load base revision

### ISSUE THREAD

**Issue by schar** [GitLiveApp/GitLive#77]
Attempting to collaborate on a project. Gitlive works intermittently, but some of the time, seemingly without rhyme or reason, we start getting errors "Failed to load base revision of [file]". Occurs for some files in project directory, not all.

**Comment by FredCoen** [comment]
Hi, @schar thanks for reporting. Have you all updated to version plugin version 14.0? If one of you hasn't you should not be able to codeshare in the first place and get the errors you mentioned above. Can you try again and report if the error persists?

With regards to the commands not being found, we will investigate and get back to you!

**Comment by schar** [comment]
Yes, we both have 14.0, though this was happening before that update as well.

**Comment by nbransby** [comment]
@schar where is this message appearing? is it when trying to view a diff of your teammate's changes?

**Comment by schar** [comment]
@nbransby Yes

**Comment by nbransby** [comment]
@schar I believe the "Failed to load base revision of [file]" issue has been fixed but the other issue you mention: "command 'git.live.showAccountList' not found", and "command 'git.live.viewSettings' not found", I have never seen before, are you still getting this?

**Comment by schar** [comment]
no, but to be frank, the unreliability forced us to find other tools
